paco-classifier-service/main.py

```
@@ -240,41 +265,59 @@ def _run():
-            status_code=500,
-            detail=f"classifier output shape {label_map.shape[:2]} != input {img.shape[:2]}",
-        )
 
-    background_png = _layer_to_rgba_png(img, label_map, BACKGROUND_LABEL)
-    stafflines_png = _layer_to_rgba_png(img, label_map, STAFFLINES_LABEL)
-    return {
-        "background_png_base64": base64.b64encode(background_png).decode(),
-        "stafflines_png_base64": base64.b64encode(stafflines_png).decode(),
-    }
\ No newline at end of file
+    async def _stream():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_stream`

(ANN202)